### PR TITLE
[#142] Refactored code and generalized formatting within expense breakdowns

### LINKED ISSUE #142: Expense breakdown total row formatting fix
There is an issue with the formatting of the total calculation section of the expense breakdown cell borders not being the right color.

### PR DESCRIPTION
**Pull request by ChaseStauts**
closes #142

### CODE REVIEW

**Comment by BreeLatimer** [comment]
Is there a task to link this PR to?

**Comment by ChaseStauts** [comment]
> Is there a task to link this PR to?
> 
> Edit: Could you add the [#142] tag to the start of the PR title so that the issues are linked :D

I tried, don't believe that's possible in the PRT title

**Review by BreeLatimer — APPROVED** [review]

File: src/jvmMain/kotlin/processing/TotalCalculations.kt [hunk 1]
```diff
@@ -53,33 +29,39 @@ fun totalCalculations(wb: Workbook, topCell: Int, bottomCell: Int, merchCol: Int
     val servCell = totalRow.createCell(8)
     val travelCell = totalRow.createCell(9)
 
+    val itemStyle = sheet.getRow(4).getCell(5).cellStyle
```
Comment (PaulEllis2001) [inline src/jvmMain/kotlin/processing/TotalCalculations.kt:32]: Have you tried this? I think that you might get some errors doing the getRow() here. I think you have to use rowIterator() or just a regular iterator over the rows.

File: src/jvmTest/kotlin/TotalDataCalcTests.kt [hunk 2]
```diff
@@ -34,17 +32,15 @@ class TotalDataCalcTests {
...
         file.copyTo(tempFile, true)
-        //Intiate streams and workbook info
+        // Intiate streams and workbook info
         fIP = java.io.FileInputStream(tempFile)
         wb = XSSFWorkbook(fIP)
         sheet = wb.getSheetAt(0)
-        dataFormat = wb.createDataFormat()
-        dollarStyle = wb.createCellStyle()
-        dollarStyle.dataFormat = dataFormat.getFormat("$#,#0.00")
+        totalStyle = sheet.getRow(5).getCell(6).cellStyle
```
Comment (PaulEllis2001) [inline src/jvmTest/kotlin/TotalDataCalcTests.kt:43]: Same thing as above here, does this give you errors? I thought that we were blocked from getting rows directly and we had to iterate over them.

**Review by PaulEllis2001 — COMMENTED** [review]
Take a look at the comments I left about getRow().

**Comment by ChaseStauts** [comment]
> Take a look at the comments I left about getRow().

I tested to make sure the implementation worked with our provided template and my test suite

**Review by PaulEllis2001 — APPROVED** [review]

**Pull request merged**